### DISCUSSION TITLE
Ordering no longer alphabetic?

### DISCUSSION THREAD

**Discussion by jordibruin** [mattcorey/indie-dev-sales#102] [General]
I thought we had to order it alphabetically by developer name, just checking since it doesn't seem like the case anymore. 🙂

**Comment by mattcorey** [comment]
It originally was, but when we got over 50 apps listed or so, we re-ordered by Category, and then by developer name, so it was a bit easier to find groups of similar apps.

BTW - I made a comment to your recent PR -- I think the 'Website' link is formatted incorrectly.  Just want to make sure you notice
